docs/UserGuide.md

```diff
@@ -229,15 +230,15 @@ Examples:
 *  `update 1 u/new 600000` Creates a new status with amount 600000 for the 1st property.
 *  `update 3 u/proceed` Moves the status of the 3rd property to next one.
```

Comment:
Don't think the user will know what does status refer to

Reply:
@dvdweien to update this section too